index.html

```diff
@@ -6,21 +6,42 @@
           content="width=device-width">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Portfolio</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/4e96f49cc5.js" crossorigin="anonymous"></script>
 </head>
 <body>
-    <nav>
+    <header>
```

Comment:
@rebaz36 great work implementing the changes requested.

Reply:
thanks @danmainah